attractions/rollup.config.js

```diff
@@ -77,6 +80,33 @@ export default [
         preprocess: [prependTagOption(icons), sveltePreprocess],
       }),
       resolve(),
+      sveld(),
+      {
+        name: 'utilsTypings',
+        writeBundle() {
+          // Note that this fails silently on Windows due to poor glob support
+          //  which is strange because it works in tsconfig's `include` option
+          spawnSync(
+            'tsc',
+            [
+              'utils/*.js',
+              '--declaration',
+              '--allowJs',
+              '--emitDeclarationOnly',
+              '--outDir',
+              'types',
+            ],
+            { shell: true }
+          );
+          // Since writeBundle hook is parallel, we need to wait for sveld to finish first
+          setTimeout(() => {
+            appendFileSync(
+              './types/index.d.ts',
+              'export * as utils from "./utils";\n'
+            );
+          }, 1000);
```

Comment:
Yes. Should I increase it to 2 seconds?
The problem is that sveld uses the `writeBundle` hook, which is the last hook to run, and also runs in parallel instead of sequentially, so I couldn't find a cleaner solution.
Of course, another option is to remove this "plugin" entirely and just add the following 2 lines to the CI, but then types will not be available when building locally. They aren't available anyways on Windows due to some stupid bug with globbing, but I thought why limit you (Linux user)?
```bash
yarn tsc utils/*.js --declaration --allowJs --emitDeclarationOnly --outDir types/utils
echo "export * as utils from './utils';" >> types/index.d.ts
```

Reply:
Scratch that, even 2 seconds is not deterministic enough.
There is a few options here:
1. Add more time
2. Move these 2 lines to the CI
3. Move them to a node script (so that it becomes one line instead of 2 from the CLI)
4. Move sveld to the first build and the custom "plugin" to the second one. Note that it seems that the builds run sequentially, even though the documentation states that they run in parallel

Note that, in conjunction, with the above methods, we can create a `tsconfig.json` to make it work cross-platform and not just on Linux.
What do you think?